ClosedXML/Excel/Caching/XLAlignmentRepository.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ClosedXML.Excel.Caching
+{
+    internal sealed class XLAlignmentRepository : XLRepositoryBase<XLAlignmentKey, XLAlignmentValue>
+    {
+        #region constructors
+        public XLAlignmentRepository(Func<XLAlignmentKey, XLAlignmentValue> createNew) : base(createNew)
```

Comment:
Just a small stylistic request, please. Put the `: base ...` on a new line. For the other classes too. It's easy to miss it if it's so far to the right.